src/components/Book.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from './Button';
+
+const Book = ({ title }) => (
+  <div>
+    <h3>{title}</h3>
+    <Button name="Remove" />
+  </div>
```

Comment:
- [ ]  The book component should accept the author name as a second argument. The author name should be rendered below the title, in a <p> tag. The author name is an important part of a book. It helps users identify the book and find it in a search. If the book component doesn't accept the author name, users will not be able to see it, which could lead to confusion. I appreciate the work you've done on the book component. I think it's a great start, but I think it would be even better if it accepted the author name. Here's an example of how you could update the book component to accept the author name:

```js
import React from 'react';

const Book = ({ title, author }) => {
  return (
    <div>
      <h1>{title}</h1>
      <p>By {author}</p>
     <Button name="Remove" />
    </div>
  );
};

export default Book;

```

Reply:
well resolved! ✔️